src/components/AuthorAbout/AuthorAbout.jsx

```diff
@@ -0,0 +1,52 @@
+import PropTypes from "prop-types";
+import "./AuthorAbout.scss";
+import fb from "../../assets/images/fb.png";
+import twitter from "../../assets/images/twitter.png";
+import insta from "../../assets/images/insta.png";
+import youtube from "../../assets/images/youtube.png";
```

Comment:
Could you please config absolute path? means instead of writing `../../assets/images/youtube.png` like this, write in this way`assets/images/youtube.png`